src/tyr/route_serializer.cc

```diff
@@ -526,7 +527,8 @@ namespace {
         const auto& current_edge = path_leg->node(idx).edge();
         bool new_name = maneuver.type() == odin::TripDirections_Maneuver_Type_kContinue ||
                         maneuver.type() == odin::TripDirections_Maneuver_Type_kBecomes;
-        bool roundabout = false;
+        bool roundabout = maneuver.type() == odin::TripDirections_Maneuver_Type_kRoundaboutEnter ||
+                          maneuver.type() == odin::TripDirections_Maneuver_Type_kRoundaboutExit;
```

Comment:
is this roundabout boolean used anymore?

Reply:
yeah not really needed.